.docker/docker-compose.yaml

```diff
@@ -38,8 +38,5 @@ services:
     container_name: mongoDB
     image: mongo
     restart: always
-    environment:
-      MONGO_INITDB_ROOT_USERNAME: admin
-      MONGO_INITDB_ROOT_PASSWORD: password
```

Comment:
Tell Kostya, that he should do the same on his branch.
@kostya4201337 
Check this PR in general, even if it's merged. It would be helpful for you